src/lib.rs

```diff
@@ -85,7 +85,23 @@ pub trait GenerateVerifiable {
 		who: Self::Member,
 		lookup: impl Fn(usize) -> Result<Self::StaticChunk, ()>,
 	) -> Result<(), ()>;
-	// push_members
+
+	/// Introduce a set of new `Member`s into the intermediate value used to build a new `Members`
+	/// value. This function is especially suitable for low domain sizes, as the provided `chunks`
+	/// must be the whole set of chunks available for a domain, such that a lookup `chunks[i]` would
+	/// yield the corresponding chunk for the `i`th key in the intermediate.
```

Comment:
AFAICT to push the member i we only need the chunk for i. Why don't we ask for the this chunk only instead of requiring the whole set of chunks?

Maybe I miss something?

Reply:
When I wrote the function at first, it wasn't obvious to me that for the `i`th member we would need exactly the `i`th stored chunk in a static chunk list all the time, for all implementations. This way of writing it basically ensured the caller wouldn't have to ever iterate over the list and gather the relevant chunks before calling this function, regardless of the underlying implementation.